### ISSUE TITLE
Clearer Advanced config

### ISSUE THREAD

**Issue by Vartkat** [FutureTense/keymaster#337]
**Is your feature request related to a problem? Please describe.**
The Custom week days are supposed to be within the Use date range and that's not clear in the interface. I first (and I think others can do it) understood that was exclusive, i.e. Use date range or Use week days

**Describe the solution you'd like**
I know it's difficult to make it clear in the interface but there could be two helpfull things : 
1- When hours are set for the Date range these hours should automtically go to each day as there can't be a day with a longer range than the one for the Date range
2- Just put a notice - replace “Custom week days" by "Custom week days (within date range)"

**Describe alternatives you've considered**
see solution suggested

**Additional context**
none

**Comment by tykeal** [comment]
I agree that the `Custom week days` does have to fall in the date range if one is defined. The two are, however, separate options that can interact in complex ways. They can be used independently _or_ together. Additionally of v0.1.x they can be individually enabled or disabled globally for the given lock in the configuration. I'm not certain that there is more that we can do to make it more clear that they are both independent _and_ interdependent at the same time.
